### PR TITLE
Restructure project (closes #10)

### LINKED ISSUE #10: Clean up project structure
I have to say I don't like the structure as it is, now.
The `app` folder is unnecessary, because everything else should probably also belong to `app`. And I also don't like it to first have to `cd app/` and only do `qmake ; make`, instead of directly do `qmake ; make`.

However I would suggest this structure:
* `bin` - Binaries (should be created after building)
* `doc` - Documentation / Licenses
    * `license_gpl-3.0.txt`
    * `license_cc_by-sa-4.0.txt`
* `src` - Source Code
    * `qml` QML-files
        * `*.qml`
    * `*.cpp` `*.h`
* `data`
    * `images`
        * `*.png` `*.svg`
    * `sounds`
        * `*.opus` `*.flac`
    * `fonts`
        * `*.ttf`
* `util` - Utilities (travis-ci scripts, deb/rpm package files, ...)
* `misc` - Miscellaneous (for logos, .desktop-files, ...)
* `README.md`
* `LICENSE.txt`
* `kaidan.pro`
* `kaidan.qrc`
* `deployment.pri`

I also like [this](https://github.com/KDE/discover) structure, but I think the one above might be better for our project.

What do you think about this?

### PR DESCRIPTION
**Pull request by LNJ2**
I also had to adjust the README and of course the qrc-file.
In the qrc-file I adjusted the path and added an alias, so
all code remains compatible (we should add such aliases for
all files in there).
And it builds now to bin/kaidan.

### CODE REVIEW

**Comment by LNJ2** [comment]
@JBBgameich No, it's not like in discover. (dicover has a folder for every lib / extension / ...)

File: kaidan.pro [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # path to local compiled swift 3 lib
-SWIFT3PATH = ../../swift-3.0
+SWIFT3PATH = ../swift-3.0
```
Comment (MRZA-MRZA) [inline kaidan.pro]: Could you explain why you've changed it?

Comment (LNJ2) [inline kaidan.pro]: `kaidan.pro` is now on the top level of our repo.

Comment (MRZA-MRZA) [inline kaidan.pro]: Sorry, haven't noticed it.

Comment (LNJ2) [inline kaidan.pro]: np

**Comment by LNJ2** [comment]
works

**Pull request merged**